Exiled.DedicatedNetwork/Host.cs

```diff
@@ -0,0 +1,369 @@
+// -----------------------------------------------------------------------
+// <copyright file="Host.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.DedicatedNetwork
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Sockets;
+    using System.Reflection;
+    using System.Threading.Tasks;
+
+    using Exiled.Network;
+    using Exiled.Network.API;
+    using Exiled.Network.API.Attributes;
+    using Exiled.Network.API.Interfaces;
+    using Exiled.Network.API.Models;
+    using Exiled.Network.API.Packets;
+
+    using LiteNetLib;
+    using LiteNetLib.Utils;
+
+    /// <summary>
+    /// Dedicated host.
+    /// </summary>
+    public class Host : NPManager, INetEventListener
+    {
+        private DedicatedConfig config;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Host"/> class.
+        /// </summary>
+        public Host()
+        {
+            Logger = new ConsoleLogger();
+
+            if (!File.Exists("./config.yml"))
```

Comment:
I'm fairly sure this will break something on Windows servers. Why not just use File.Exists("config.yml") ? File.Exists always defaults to checking the working directory of the exe/library, which is the same as ./ in Linux.